docker/.qa.env

```diff
@@ -1,6 +1,8 @@
 # these hosts will likely be rewritten for the relevant QA merge id details
 API_HOST=http://qa.api.spira.io:8080
 APP_HOST=http://qa.app.spira.io
+FORUM_HOST=http://qa.forum.spira.io:8008
```

Comment:
We can't actually use these variables in qa - I need to revisit how they're being uses as the QA environment doesn't support ports other than 80 (our nginx proxy needs to be used).

You will need to use the `HOSTNAME_FORUM` as that is compatible with the qa environment.